pkg/iut/v1alpha1/v1alpha1.go

```diff
@@ -166,43 +177,50 @@ func (h V1Alpha1Handler) Start(w http.ResponseWriter, r *http.Request, ps httpro
 
 // Status creates a simple DONE Status response with IUTs.
 func (h V1Alpha1Handler) Status(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
-	identifier := r.Header.Get("X-Etos-Id")
+	identifier, err := uuid.Parse(r.Header.Get("X-Etos-Id"))
+	if err != nil {
+		RespondWithError(w, http.StatusInternalServerError, err.Error())
+	}
 	logger := h.logger.WithField("identifier", identifier).WithContext(r.Context())
 
 	id, err := uuid.Parse(r.URL.Query().Get("id"))
+	client := h.database.Open(r.Context(), identifier)
+
+	data := make([]byte, 4096)
```

Comment:
Is 4096 always goign to be enough here?

Reply:
make([]byte) should be fine here since we cannot know the size

Reply:
`make([]byte)` will not compile. It has to be a list of bytes that is larger than the actual data.

Reply:
Then I would recommend using something `io.ReadAll` instead, which will allocate a small size and grow it when necessary.